eventActions/tosReminderAction.js

```diff
@@ -0,0 +1,63 @@
+const Discord = require('discord.js')
+table = require('../databaseFiles/tosReminderTable');
+
+const addToDatabase = async function(user,joinTime,reminded=false){
+    table.sync(/*{ force: true }*/).then(async ()=>{
+        await table.create({
+            user_id: user.id,
+            joinTime: joinTime,
+            reminded: reminded
+        }).catch(err =>{
+            console.error('Tosreminder error: ' + err )
+        })
+    })
+}
+const removeFromDatabase = async function(user_id){
+    await table.destory({
+        where:{
+            user_id:user_id
+        }
+    }).catch(err =>{
+        console.error("Tos reminder error" + err)
+    })
+}
+const tosRemind = async function(client){
+    messageEmbed = new Discord.MessageEmbed()
+    .setColor("#FC0303")
```

Comment:
For this embed, let's use the `koaYellow` color from the config. For readability, can you indent the `.setColor` and `.addField` lines by one?